src/server/node_recov_db.c

```diff
@@ -373,8 +375,15 @@ node_recov_db(void *nd)
 	return np;
 
 db_err:
+	if (np_init) {
```

Comment:
I don't see this function being called anywhere. I am not sure if this function is even needed anymore. Do you really want to keep this change?

Reply:
Thanks for noticing, never realized its so.
I have now purged dead functions node_recov_db() and static db_to_svr_node() in https://github.com/PBSPro/pbspro/pull/1308/commits/8060bef991e0d99022e782e4bd322fa21548b336

Reply:
This fact has been updated in the heading comment/ bug description